# Data Collection

The purpose of this notebook is to demonstrate how the data for the Summary Notebook was collected, in case you are interested in trying to reproduce this study in some fashion.

## Knight Commission

The first source used in this project is the [Knight Commission on Intercollegiate Athletics](https://www.knightcommission.org/). Their database is accessible [here](https://cafidatabase.knightcommission.org/). From the databse website, click on "Go to Custom Reporting."

![img](../Images/Knight_custom_reporting.png)

From there, you can customize and select any variables with which you are interested. Given that I wanted to focus on athletic spending (in particular football spending), I selected all variables for every option available on this website. Note that I selected "All Institutions," but did not select any of the NCAA or FBS variables in the section below. I also checked the box to adjust for cost inflation.

![img](../Images/Knight_data.png)

![img](../Images/Knight_schools.png)

![img](../Images/Knight_ncaa_years.png)

When you click "Generate Report" at the bottom of the page, you are then taken to a page titled "Custom Reporting Results." You can either examine the results directly on the website, or if you want to download the data, you can click "Download Data." Note that you will have to enter some personal information before you can download the excel file.

![img](../Images/Knight_custom_reporting_results.png)

Once downloaded, the Excel spreadsheet contains all of the data selected along with a data dictionary on a separate sheet. The formatting is a little odd when you try to open the original Excel sheet using pandas, given that the first three columns are frozen. You can either unfreeze the columns directly on the sheet, or you can create a new Excel documnet, copy/paste the first three rows, and then copy/paste the rest. There are a lot of missing values in this data set, but we will deal with that later when merging all three of our data sets.

In [1]:
import pandas as pd

In [2]:
knight = pd.read_excel('../Data/knight_05_19.xlsx')
knight.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 5295 entries, 0 to 5294
Data columns (total 31 columns):
 #   Column                                                                 Non-Null Count  Dtype  
---  ------                                                                 --------------  -----  
 0   Data                                                                   5295 non-null   object 
 1   IPEDS ID                                                               5295 non-null   int64  
 2   Year                                                                   5295 non-null   int64  
 3   NCAA Subdivision                                                       5295 non-null   object 
 4   FBS Conference                                                         1853 non-null   object 
 5   Total Expenses                                                         3370 non-null   float64
 6   Excess Transfers Back                                                  3370 non-null   f

## Equity in Athletics Data Analysis (EADA)

The next source used in this project is the [Equity in Athletics Data Analysis website](https://ope.ed.gov/athletics/#/). From the homepage, click on "Download Custom Data." On this page you can choose any number of institutions you would like, but for my project I clicked "Select All" so that I could get data on as many colleges as possible.

![img](../Images/EADA_institutions.png)

On the next page, I selected years between 2005–2019 to be consistent with the Knight Commission data, but as you can see they also have data from 2003–2003 available. 

![img](../Images/EADA_years.png)

There are a lot of options for downloading data on this website, but for the purposes of my project this is what I selected:
- Athletics Participation - All Sports and Men's, Women's and Coed Teams
    - Contains the number of student athletes at each institution
- Head Coaches - Men's Team - All Sports and Head Coaches University Employee
    - Contains the number of head coaches for every sport at each institution. 
- Assistant Coaches - Men's Team - All Sports and Assistant Coaches University Employee
    - Contains the number of assistant coaches for every sport at each institution.
- Revenues - All Sports and Men's, Women's and Coed Teams
    - Contains the amount of revenue for all teams at each institution.
- Expenses - All Sports and Men's, Women's and Coed Teams
    - Contains the amount of expenses for all teams at each institution.

![img](../Images/EADA_numbers.png)

![img](../Images/EADA_expenses.png)

If you select multiple options like I did, when you click "Download" at the bottom of the page you will download a ZIP file that contains a CSV file for each option you selected. I will go through each of these five files here in turn so you can see how I got to the final version of the file in my summary notebook.

### Student Athletes
We can see this file contains 209 columns with information about participation for all athletics programs at each institution. However, for my project I was only concerned with the total number of student athletes. Thus, I created a new dataframe that just has 'Unduplicated Count Men's Participation' and 'Unduplicated Count Women's Participation', along with the first four columns containing information about the year and school.

In [3]:
stu_ath = pd.read_csv('../Data/EADA_stud_ath_all_years.csv')

In [4]:
stu_ath.head()

Survey Year  UNITID                      Institution Name State CD  \
0         2005  222178          Abilene Christian University       TX   
1         2005  138558  Abraham Baldwin Agricultural College       GA   
2         2005  126182                   Adams State College       CO   
3         2005  188429                    Adelphi University       NY   
4         2005  188438          Adirondack Community College       NY   

                   Classification Name Classification Other  Sanction Code  \
0     NCAA Division II (with football)                  NaN              1   
1                     NJCAA Division I                  NaN              3   
2     NCAA Division II (with football)                  NaN              1   
3  NCAA Division II (without football)                  NaN              1   
4                   NJCAA Division III                  NaN              3   

  Sanction Name  Male Undergraduates  Female Undergraduates  ...  \
0          NCAA               1792.0                 2137.0  ...   
1         NJCAA               1103.0                 1134.0  ...   
2          NCAA                851.0                  929.0  ...   
3          NCAA               1155.0                 2806.0  ...   
4         NJCAA                874.0                 1141.0  ...   

   Other Sports Coed Team Women Participation  \
0                                         NaN   
1                                         NaN   
2                                         NaN   
3                                         NaN   
4                                         NaN   

   Other Sports Total Participation  Total Men's Team Participation  \
0                               NaN                           207.0   
1                               NaN                            58.0   
2                               NaN                           272.0   
3                               NaN                           156.0   
4                               NaN                            62.0   

   Total Women's Team Participation  Total Coed Team Men Participation  \
0                              89.0                                NaN   
1                              40.0                                NaN   
2                             114.0                                NaN   
3                             137.0                                NaN   
4                              34.0                                NaN   

   Total Coed Team Women Participation  Grand Total Men's Participation  \
0                                  NaN                              207   
1                                  NaN                               58   
2                                  NaN                              272   
3                                  NaN                              156   
4                                  NaN                               62   

   Grand Total Women's Participation  Unduplicated Count Men's Participation  \
0                                 89                                   207.0   
1                                 40                                    58.0   
2                                114                                   264.0   
3                                137                                   145.0   
4                                 34                                    62.0   

   Unduplicated Count Women's Participation  
0                                      89.0  
1                                      40.0  
2                                     110.0  
3                                     105.0  
4                                      30.0  

[5 rows x 209 columns]

In [5]:
stu_ath = stu_ath[['Survey Year', 'UNITID', 'Institution Name', 'Classification Name', 'Unduplicated Count Men\'s Participation', 'Unduplicated Count Women\'s Participation']]
                   

I then renamed the last two columns so that I would not have to deal with apostrophes.

In [6]:
stu_ath.rename(columns={'Unduplicated Count Men\'s Participation': 'Total Male Participation',
                       'Unduplicated Count Women\'s Participation': 'Total Female Participation'},
              inplace=True)

stu_ath

Survey Year  UNITID                      Institution Name  \
0             2005  222178          Abilene Christian University   
1             2005  138558  Abraham Baldwin Agricultural College   
2             2005  126182                   Adams State College   
3             2005  188429                    Adelphi University   
4             2005  188438          Adirondack Community College   
...            ...     ...                                   ...   
29441         2019  181853                          York College   
29442         2019  217059          York College of Pennsylvania   
29443         2019  141361                  Young Harris College   
29444         2019  206695           Youngstown State University   
29445         2019  126119                          Yuba College   

                       Classification Name  Total Male Participation  \
0         NCAA Division II (with football)                     207.0   
1                         NJCAA Division I                      58.0   
2         NCAA Division II (with football)                     264.0   
3      NCAA Division II (without football)                     145.0   
4                       NJCAA Division III                      62.0   
...                                    ...                       ...   
29441                     NAIA Division II                     166.0   
29442   NCAA Division III without football                     259.0   
29443    NCAA Division II without football                     191.0   
29444                  NCAA Division I-FCS                     260.0   
29445                                CCCAA                     134.0   

       Total Female Participation  
0                            89.0  
1                            40.0  
2                           110.0  
3                           105.0  
4                            30.0  
...                           ...  
29441                       146.0  
29442                       221.0  
29443                       123.0  
29444                       187.0  
29445                        55.0  

[29446 rows x 6 columns]

### Head Coaches
We can see this file contains 207 columns with information about head coaches for all athletics programs at each institution. However, for my project I was only concerned with the total number of head football coaches. Thus, I created a new dataframe that just has 'Football Total', along with the first four columns containing information about the year and school. I determined that this was the column I needed by creating a list comprehension to find columns that had 'Football' in the name, and then selected the one column that represents the total number of head football coaches at each institution.

In [7]:
hc = pd.read_csv('../Data/EADA_head_coaches_all_years.csv')
hc.head()

Survey Year  UNITID                      Institution Name State CD  \
0         2005  222178          Abilene Christian University       TX   
1         2005  138558  Abraham Baldwin Agricultural College       GA   
2         2005  126182                   Adams State College       CO   
3         2005  188429                    Adelphi University       NY   
4         2005  188438          Adirondack Community College       NY   

                   Classification Name Classification Other  Sanction Code  \
0     NCAA Division II (with football)                  NaN              1   
1                     NJCAA Division I                  NaN              3   
2     NCAA Division II (with football)                  NaN              1   
3  NCAA Division II (without football)                  NaN              1   
4                   NJCAA Division III                  NaN              3   

  Sanction Name  Male Undergraduates  Female Undergraduates  ...  \
0          NCAA               1792.0                 2137.0  ...   
1         NJCAA               1103.0                 1134.0  ...   
2          NCAA                851.0                  929.0  ...   
3          NCAA               1155.0                 2806.0  ...   
4         NJCAA                874.0                 1141.0  ...   

   Other Sports Male HD Coach FT Univ. Emp.  \
0                                       NaN   
1                                       NaN   
2                                       NaN   
3                                       NaN   
4                                       NaN   

  Other Sports Male HD Coach PT Univ. Emp.  \
0                                      NaN   
1                                      NaN   
2                                      NaN   
3                                      NaN   
4                                      NaN   

   Other Sports Female HD Coach FT Univ. Emp.  \
0                                         NaN   
1                                         NaN   
2                                         NaN   
3                                         NaN   
4                                         NaN   

   Other Sports Female HD Coach PT Univ. Emp.  Other Sports Total  \
0                                         NaN                 NaN   
1                                         NaN                 NaN   
2                                         NaN                 NaN   
3                                         NaN                 NaN   
4                                         NaN                 NaN   

   Total Male HD Coach FT Univ. Emp.  Total Male HD Coach PT Univ. Emp.  \
0                                  4                                  3   
1                                  3                                  1   
2                                  4                                  1   
3                                  5                                  3   
4                                  3                                  2   

   Total Female HD Coach FT Univ. Emp.  Total Female HD Coach PT Univ. Emp.  \
0                                    0                                    0   
1                                    0                                    0   
2                                    0                                    0   
3                                    0                                    1   
4                                    0                                    0   

   Total HD Coaches  
0                 7  
1                 4  
2                 5  
3                 9  
4                 5  

[5 rows x 207 columns]

In [8]:
hc_fb_cols = [col for col in hc.columns if 'Football' in col]
hc_fb_cols

['Football Male HD Coach FT Univ. Emp.',
 'Football Male HD Coach PT Univ. Emp.',
 'Football Female HD Coach FT Univ. Emp.',
 'Football Female HD Coach PT Univ. Emp.',
 'Football Total']

In [9]:
hc = hc[['Survey Year', 'UNITID', 'Institution Name', 'Classification Name', 'Football Total']]

I then renamed this column 'HC Total' to be more descriptive.

In [10]:
hc.rename(columns={'Football Total': 'HC Total'},
         inplace=True)

hc

Survey Year  UNITID                      Institution Name  \
0             2005  222178          Abilene Christian University   
1             2005  138558  Abraham Baldwin Agricultural College   
2             2005  126182                   Adams State College   
3             2005  188429                    Adelphi University   
4             2005  188438          Adirondack Community College   
...            ...     ...                                   ...   
29207         2019  181853                          York College   
29208         2019  217059          York College of Pennsylvania   
29209         2019  141361                  Young Harris College   
29210         2019  206695           Youngstown State University   
29211         2019  126119                          Yuba College   

                       Classification Name  HC Total  
0         NCAA Division II (with football)       1.0  
1                         NJCAA Division I       NaN  
2         NCAA Division II (with football)       1.0  
3      NCAA Division II (without football)       NaN  
4                       NJCAA Division III       NaN  
...                                    ...       ...  
29207                     NAIA Division II       NaN  
29208   NCAA Division III without football       NaN  
29209    NCAA Division II without football       NaN  
29210                  NCAA Division I-FCS       1.0  
29211                                CCCAA       1.0  

[29212 rows x 5 columns]

### Assistant Coaches
This process was pretty similar to the head coaches steps outlined above. We can see this file contains 207 columns with information about assistant coaches for all athletics programs at each institution. However, for my project I was only concerned with the total number of assistant football coaches. Thus, I created a new dataframe that just has 'Football Total', along with the first four columns containing information about the year and school. I determined that this was the column I needed by creating a list comprehension to find columns that had 'Football' in the name, and then selected the one column that represents the total number of head football coaches at each institution.

In [11]:
ac = pd.read_csv('../Data/EADA_asst_coaches_all_years.csv')
ac.head()

Survey Year  UNITID                      Institution Name State CD  \
0         2005  222178          Abilene Christian University       TX   
1         2005  138558  Abraham Baldwin Agricultural College       GA   
2         2005  126182                   Adams State College       CO   
3         2005  188429                    Adelphi University       NY   
4         2005  188438          Adirondack Community College       NY   

                   Classification Name Classification Other  Sanction Code  \
0     NCAA Division II (with football)                  NaN              1   
1                     NJCAA Division I                  NaN              3   
2     NCAA Division II (with football)                  NaN              1   
3  NCAA Division II (without football)                  NaN              1   
4                   NJCAA Division III                  NaN              3   

  Sanction Name  Male Undergraduates  Female Undergraduates  ...  \
0          NCAA               1792.0                 2137.0  ...   
1         NJCAA               1103.0                 1134.0  ...   
2          NCAA                851.0                  929.0  ...   
3          NCAA               1155.0                 2806.0  ...   
4         NJCAA                874.0                 1141.0  ...   

   Other Sports Male Assistant Coach FT Univ. Emp.  \
0                                              NaN   
1                                              NaN   
2                                              NaN   
3                                              NaN   
4                                              NaN   

  Other Sports Male Assistant Coach PT Univ. Emp.  \
0                                             NaN   
1                                             NaN   
2                                             NaN   
3                                             NaN   
4                                             NaN   

   Other Sports Female Assistant Coach FT Univ. Emp.  \
0                                                NaN   
1                                                NaN   
2                                                NaN   
3                                                NaN   
4                                                NaN   

   Other Sports Female Assistant Coach PT Univ. Emp.  Other Sports Total  \
0                                                NaN                 NaN   
1                                                NaN                 NaN   
2                                                NaN                 NaN   
3                                                NaN                 NaN   
4                                                NaN                 NaN   

   Total Male Assistant Coach FT Univ. Emp.  \
0                                         9   
1                                         0   
2                                         6   
3                                         1   
4                                         0   

   Total Male Assistant Coach PT Univ. Emp.  \
0                                         4   
1                                         1   
2                                         3   
3                                        17   
4                                         3   

   Total Female Assistant Coach FT Univ. Emp.  \
0                                           0   
1                                           0   
2                                           0   
3                                           1   
4                                           0   

   Total Female Assistant Coach PT Univ. Emp.  Total Assistant Coaches  
0                                           1                       14  
1                                           0                        1  
2                                           0                        9  
3                                           2                       21  
4                     

In [12]:
ac_fb_cols = [col for col in ac.columns if 'Football' in col]
ac_fb_cols

['Football Male Assistant Coach FT Univ. Emp.',
 'Football Male Assistant Coach PT Univ. Emp.',
 'Football Female Assistant Coach FT Univ. Emp.',
 'Football Female Assistant Coach PT Univ. Emp.',
 'Football Total']

In [13]:
ac = ac[['Survey Year', 'UNITID', 'Institution Name', 'Classification Name', 'Football Total']]

Like above, I renamed this column 'AC Total' to be more descriptive

In [14]:
ac.rename(columns={'Football Total': 'AC Total'},
         inplace=True)

ac

Survey Year  UNITID                      Institution Name  \
0             2005  222178          Abilene Christian University   
1             2005  138558  Abraham Baldwin Agricultural College   
2             2005  126182                   Adams State College   
3             2005  188429                    Adelphi University   
4             2005  188438          Adirondack Community College   
...            ...     ...                                   ...   
28509         2019  181853                          York College   
28510         2019  217059          York College of Pennsylvania   
28511         2019  141361                  Young Harris College   
28512         2019  206695           Youngstown State University   
28513         2019  126119                          Yuba College   

                       Classification Name  AC Total  
0         NCAA Division II (with football)       7.0  
1                         NJCAA Division I       NaN  
2         NCAA Division II (with football)       6.0  
3      NCAA Division II (without football)       NaN  
4                       NJCAA Division III       NaN  
...                                    ...       ...  
28509                     NAIA Division II       NaN  
28510   NCAA Division III without football       NaN  
28511    NCAA Division II without football       NaN  
28512                  NCAA Division I-FCS      10.0  
28513                                CCCAA       4.0  

[28514 rows x 5 columns]

### Revenues
We can see this file contains 168 columns with information about revenues for all athletics programs at each institution. However, for my project I was only concerned with the total revenue. Thus, I created a new dataframe that just has the last column 'Grand Total Revenue', along with the first four columns containing information about the year and school.

In [15]:
rev = pd.read_csv('../Data/EADA_revenues_all_years.csv')
rev.head()

Survey Year  UNITID                      Institution Name State CD  \
0         2005  222178          Abilene Christian University       TX   
1         2005  138558  Abraham Baldwin Agricultural College       GA   
2         2005  126182                   Adams State College       CO   
3         2005  188429                    Adelphi University       NY   
4         2005  188438          Adirondack Community College       NY   

                   Classification Name Classification Other  Sanction Code  \
0     NCAA Division II (with football)                  NaN              1   
1                     NJCAA Division I                  NaN              3   
2     NCAA Division II (with football)                  NaN              1   
3  NCAA Division II (without football)                  NaN              1   
4                   NJCAA Division III                  NaN              3   

  Sanction Name  Male Undergraduates  Female Undergraduates  ...  \
0          NCAA               1792.0                 2137.0  ...   
1         NJCAA               1103.0                 1134.0  ...   
2          NCAA                851.0                  929.0  ...   
3          NCAA               1155.0                 2806.0  ...   
4         NJCAA                874.0                 1141.0  ...   

   Wrestling Total Revenue  Other Sports Men's Team Revenue  \
0                      NaN                              NaN   
1                      NaN                              NaN   
2                 163368.0                              NaN   
3                      NaN                              NaN   
4                      NaN                              NaN   

   Other Sports Women's Team Revenue  Other Sports Coed Team Revenue  \
0                                NaN                             NaN   
1                                NaN                             NaN   
2                                NaN                             NaN   
3                                NaN                             NaN   
4                                NaN                             NaN   

   Other Sports Total Revenue  Total Men's Team Revenue  \
0                         NaN                 2934983.0   
1                         NaN                  307008.0   
2                         NaN                 1447101.0   
3                         NaN                 1650217.0   
4                         NaN                   12914.0   

   Total Women's Team Revenue  Total Coed Team Revenue  \
0                   1632933.0                      NaN   
1                    282614.0                      NaN   
2                   1080955.0                      NaN   
3                   1407314.0                      NaN   
4                      4940.0                      NaN   

   Not Allocated by Gender/Sport Revenue  Grand Total Revenue  
0                               832469.0              5400385  
1                                53675.0               643297  
2                               253607.0              2781663  
3                              1192266.0              4249797  
4                                96986.0               114840  

[5 rows x 168 columns]

In [16]:
rev = rev[['Survey Year', 'UNITID', 'Institution Name', 'Classification Name', 'Grand Total Revenue']]
rev

Survey Year  UNITID                      Institution Name  \
0             2005  222178          Abilene Christian University   
1             2005  138558  Abraham Baldwin Agricultural College   
2             2005  126182                   Adams State College   
3             2005  188429                    Adelphi University   
4             2005  188438          Adirondack Community College   
...            ...     ...                                   ...   
29441         2019  181853                          York College   
29442         2019  217059          York College of Pennsylvania   
29443         2019  141361                  Young Harris College   
29444         2019  206695           Youngstown State University   
29445         2019  126119                          Yuba College   

                       Classification Name  Grand Total Revenue  
0         NCAA Division II (with football)              5400385  
1                         NJCAA Division I               643297  
2         NCAA Division II (with football)              2781663  
3      NCAA Division II (without football)              4249797  
4                       NJCAA Division III               114840  
...                                    ...                  ...  
29441                     NAIA Division II              3805596  
29442   NCAA Division III without football              2857778  
29443    NCAA Division II without football              5751311  
29444                  NCAA Division I-FCS             16292051  
29445                                CCCAA               743969  

[29446 rows x 5 columns]

### Expenses
This process was similar to the revenues section above. We can see this file contains 168 columns with information about expenses for all athletics programs at each institution. However, for my project I was only concerned with the total expenses. Thus, I created a new dataframe that just has the last column 'Grand Total Expenses', along with the first four columns containing information about the year and school.

In [17]:
exp = pd.read_csv('../Data/EADA_expenses_all_years.csv')
exp

Survey Year  UNITID                      Institution Name State CD  \
0             2005  222178          Abilene Christian University       TX   
1             2005  138558  Abraham Baldwin Agricultural College       GA   
2             2005  126182                   Adams State College       CO   
3             2005  188429                    Adelphi University       NY   
4             2005  188438          Adirondack Community College       NY   
...            ...     ...                                   ...      ...   
29441         2019  181853                          York College       NE   
29442         2019  217059          York College of Pennsylvania       PA   
29443         2019  141361                  Young Harris College       GA   
29444         2019  206695           Youngstown State University       OH   
29445         2019  126119                          Yuba College       CA   

                       Classification Name Classification Other  \
0         NCAA Division II (with football)                  NaN   
1                         NJCAA Division I                  NaN   
2         NCAA Division II (with football)                  NaN   
3      NCAA Division II (without football)                  NaN   
4                       NJCAA Division III                  NaN   
...                                    ...                  ...   
29441                     NAIA Division II                  NaN   
29442   NCAA Division III without football                  NaN   
29443    NCAA Division II without football                  NaN   
29444                  NCAA Division I-FCS                  NaN   
29445                                CCCAA                  NaN   

       Sanction Code Sanction Name  Male Undergraduates  \
0                  1          NCAA               1792.0   
1                  3         NJCAA               1103.0   
2                  1          NCAA                851.0   
3                  1          NCAA               1155.0   
4                  3         NJCAA                874.0   
...              ...           ...                  ...   
29441              2          NAIA                212.0   
29442              1          NCAA               1596.0   
29443              1          NCAA                433.0   
29444              1          NCAA               4048.0   
29445              9         other                917.0   

       Female Undergraduates  ...  Wrestling Total Expenses  \
0                     2137.0  ...                       NaN   
1                     1134.0  ...                       NaN   
2                      929.0  ...                  163368.0   
3                     2806.0  ...                       NaN   
4                     1141.0  ...                       NaN   
...                      ...  ...                       ...   
29441                  191.0  ...                  376715.0   
29442                 2085.0  ...                  107567.0   
29443                  468.0  ...                       NaN   
29444                 4528.0  ...                       NaN   
29445                 1290.0  ...                       NaN   

       Other Sports Men's Team Expenses  Other Sports Women's Team Expenses  \
0                                   NaN                                 NaN   
1                                   NaN                                 NaN   
2                                   NaN                                 NaN   
3                                   NaN                                 NaN   
4                                   NaN                                 NaN   
...                                 ...                                 ...   
29441                               NaN                            107334.0   
29442                               NaN                                 NaN   
29443                               NaN                                 NaN   
29444                

In [18]:
exp = exp[['Survey Year', 'UNITID', 'Institution Name', 'Classification Name', 'Grand Total Expenses']]
exp

Survey Year  UNITID                      Institution Name  \
0             2005  222178          Abilene Christian University   
1             2005  138558  Abraham Baldwin Agricultural College   
2             2005  126182                   Adams State College   
3             2005  188429                    Adelphi University   
4             2005  188438          Adirondack Community College   
...            ...     ...                                   ...   
29441         2019  181853                          York College   
29442         2019  217059          York College of Pennsylvania   
29443         2019  141361                  Young Harris College   
29444         2019  206695           Youngstown State University   
29445         2019  126119                          Yuba College   

                       Classification Name  Grand Total Expenses  
0         NCAA Division II (with football)               5074419  
1                         NJCAA Division I                643297  
2         NCAA Division II (with football)               2781663  
3      NCAA Division II (without football)               3556017  
4                       NJCAA Division III                111670  
...                                    ...                   ...  
29441                     NAIA Division II               3805596  
29442   NCAA Division III without football               2638512  
29443    NCAA Division II without football               5751311  
29444                  NCAA Division I-FCS              16292051  
29445                                CCCAA                743969  

[29446 rows x 5 columns]

### Merging EADA
Now that we have the data needed for this project, all that is left to do is merge all five of these data frames. For each merge, I joined on the 'Survey Year', 'UNITID', 'Institution Name', and 'Classification Name' columns using an outer join to retain as much data as possible. There are still missing values after these outer joins, but I deal with those in the summary notebook so for now I am not concerned.

In [19]:
eada_merge_1 = pd.merge(rev, exp, how = 'outer',
                       left_on=['Survey Year', 'UNITID', 'Institution Name', 'Classification Name'],
                       right_on=['Survey Year', 'UNITID', 'Institution Name', 'Classification Name'])

eada_merge_2 = pd.merge(eada_merge_1, stu_ath, how = 'outer',
                       left_on=['Survey Year', 'UNITID', 'Institution Name', 'Classification Name'],
                       right_on=['Survey Year', 'UNITID', 'Institution Name', 'Classification Name'])

eada_merge_3 = pd.merge(eada_merge_2, hc, how = 'outer',
                       left_on=['Survey Year', 'UNITID', 'Institution Name', 'Classification Name'],
                       right_on=['Survey Year', 'UNITID', 'Institution Name', 'Classification Name'])

eada_final = pd.merge(eada_merge_3, ac, how = 'outer',
                       left_on=['Survey Year', 'UNITID', 'Institution Name', 'Classification Name'],
                       right_on=['Survey Year', 'UNITID', 'Institution Name', 'Classification Name'])

I then created an additional column 'Total FB Coaches' by adding together the 'HC Total' and 'AC Total' columns.

In [20]:
eada_final['Total FB Coaches'] = eada_final['HC Total'] + eada_final['AC Total']

In [21]:
eada_final

Survey Year  UNITID                      Institution Name  \
0             2005  222178          Abilene Christian University   
1             2005  138558  Abraham Baldwin Agricultural College   
2             2005  126182                   Adams State College   
3             2005  188429                    Adelphi University   
4             2005  188438          Adirondack Community College   
...            ...     ...                                   ...   
29441         2019  181853                          York College   
29442         2019  217059          York College of Pennsylvania   
29443         2019  141361                  Young Harris College   
29444         2019  206695           Youngstown State University   
29445         2019  126119                          Yuba College   

                       Classification Name  Grand Total Revenue  \
0         NCAA Division II (with football)              5400385   
1                         NJCAA Division I               643297   
2         NCAA Division II (with football)              2781663   
3      NCAA Division II (without football)              4249797   
4                       NJCAA Division III               114840   
...                                    ...                  ...   
29441                     NAIA Division II              3805596   
29442   NCAA Division III without football              2857778   
29443    NCAA Division II without football              5751311   
29444                  NCAA Division I-FCS             16292051   
29445                                CCCAA               743969   

       Grand Total Expenses  Total Male Participation  \
0                   5074419                     207.0   
1                    643297                      58.0   
2                   2781663                     264.0   
3                   3556017                     145.0   
4                    111670                      62.0   
...                     ...                       ...   
29441               3805596                     166.0   
29442               2638512                     259.0   
29443               5751311                     191.0   
29444              16292051                     260.0   
29445                743969                     134.0   

       Total Female Participation  HC Total  AC Total  Total FB Coaches  
0                            89.0       1.0       7.0               8.0  
1                            40.0       NaN       NaN               NaN  
2                           110.0       1.0       6.0               7.0  
3                           105.0       NaN       NaN               NaN  
4                            30.0       NaN       NaN               NaN  
...                           ...       ...       ...               ...  
29441                       146.0       NaN       NaN               NaN  
29442                       221.0       NaN       NaN               NaN  
29443                       123.0       NaN       NaN               NaN  
29444                       187.0       1.0      10.0              11.0  
29445                        55.0       1.0       4.0               5.0  

[29446 rows x 11 columns]

## The Integrated Postsecondary Education Data System (IPEDS)